docs/zh/latest/how-to-build.md

```diff
@@ -21,146 +21,200 @@ title: 如何构建 Apache APISIX
 
 ### 通过 Helm Chart 安装
 
-见 https://github.com/apache/apisix-helm-chart
+详情请参考：[使用 Helm Chart 安装 Apache APISIX](https://github.com/apache/apisix-helm-chart)。
 
-### 通过源码包安装
+## 步骤3：管理 APISIX 服务
 
-你需要先下载 Apache Release 源码包：
+我们可以在 APISIX 的目录下使用命令初始化依赖、启动服务和停止服务，也可以通过 `make help` 命令查看所有命令和对应的功能。
 
-```shell
-$ mkdir apisix-2.7
-$ wget https://downloads.apache.org/apisix/2.7/apache-apisix-2.7-src.tgz
-$ tar zxvf apache-apisix-2.7-src.tgz -C apisix-2.7
-```
+### 初始化依赖
 
-安装运行时依赖的 Lua 库：
+运行以下命令初始化 Nginx 配置文件和 etcd。
 
-```
-cd apisix-2.7
-make deps
+```shell
+# init nginx config file and etcd
+make init
 ```
 
-## 3. 管理（启动、关闭等）APISIX 服务
+### 启动 APISIX
 
-我们可以在 apisix 的目录下用 `make run` 命令来启动服务，或者用 `make stop` 方式关闭服务。
+运行以下命令启动 APISIX。
 
 ```shell
-# init nginx config file and etcd
-$ make init
-
 # start APISIX server
-$ make run
+make run
+```
+
+### 停止运行 APISIX
+
+优雅停机 `make quit` 和强制停机 `make stop`都可以停止运行 APISIX。建议您优先选择优雅停机的方式停止 APISIX，因为这种停止方式能够保证 APISIX 完成了已经接受到的请求之后再停止；而强制停机则是立即停止 APISIX，在这种情况下，APISIX 接收到但未完成的请求会随着强制停机一并停止。
 
+执行优雅停机的命令如下所示：
+
+```shell
 # stop APISIX server gracefully
-$ make quit
+make quit
+```
 
+执行强制停机的命令如下所示：
+
+```shell
 # stop APISIX server immediately
-$ make stop
+make stop
+```
 
+### 查看其他操作
+
+运行`make help` 命令，查看返回结果，获取其他操作的命令和描述。
+
+```shell
 # more actions find by `help`
-$ make help
+make help
 ```
 
-## 4. 运行测试案例
+## 步骤4：运行测试案例
+
+1. 安装 `perl` 的包管理器 `cpanminus`。
+
+2. 然后通过 `cpanm` 来安装 test-nginx 的依赖：
 
-1. 先安装 perl 的包管理器 cpanminus
-2. 然后通过 cpanm 来安装 test-nginx 的依赖：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
-3. 然后 clone 最新的源码：`git clone https://github.com/iresty/test-nginx.git`。注意使用我们 fork 出来的版本。
-4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集：
+  ```shell
+  sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)
+  ```
+
+3. 然后 clone 最新的源码，请使用我们 fork 出来的版本：
+
+  ```shell
+  git clone https://github.com/iresty/test-nginx.git
+  ```
+
+4. 通过 `perl` 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集：
     * 追加当前目录到perl模块目录： `export PERL5LIB=.:$PERL5LIB`
     * 直接运行：`make test`
-    * 指定 nginx 二进制路径：`TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t`
+    * 指定 Nginx 二进制路径：`TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t`
     * 部分测试需要依赖外部服务和修改系统配置。如果想要完整地构建测试环境，可以参考 `ci/linux_openresty_common_runner.sh`。
 
-### 疑难排解测试
+### 问题排查
 
 **配置 Nginx 路径**
 
-如果遇到问题 `Error unknown directive "lua_package_path" in /API_ASPIX/apisix/t/servroot/conf/nginx.conf`
-确保将openresty设置为默认的nginx并按如下所示导出路径。
+出现`Error unknown directive "lua_package_path" in /API_ASPIX/apisix/t/servroot/conf/nginx.conf` 报错的解决方法如下：
+
+确保将 Openresty 设置为默认的 Nginx，并按如下所示导出路径：
 
 * export PATH=/usr/local/openresty/nginx/sbin:$PATH
   * Linux 默认安装路径：
     * export PATH=/usr/local/openresty/nginx/sbin:$PATH
-  * OSx 通过homebrew默认安装路径：
+  * MacOS X 通过 homebrew 默认安装路径：
```

Comment:
It should be called MacOS right now
Ref: https://en.wikipedia.org/wiki/MacOS

Reply:
Got it. Fixed